src/ZLogger.Unity/Assets/Scripts/ZLogger/AsyncStreamLineMessageWriter.cs

```diff
@@ -1,62 +1,212 @@
 using System;
+using System.Buffers;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
 using System.Diagnostics;
 using System.IO;
 using System.Runtime.CompilerServices;
 using System.Text;
 using System.Threading;
 using System.Threading.Channels;
 using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+using ZLogger.Entries;
 
 namespace ZLogger
 {
     public class AsyncStreamLineMessageWriter : IAsyncLogProcessor, IAsyncDisposable
     {
         readonly byte[] newLine;
-        readonly bool crlf;
-        readonly byte newLine1;
-        readonly byte newLine2;
-
-        readonly Stream stream;
-        readonly Channel<IZLoggerEntry> channel;
-        readonly Task writeLoop;
-        readonly ZLoggerOptions options;
+        readonly bool   crlf;
+        readonly byte   newLine1;
+        readonly byte   newLine2;
+
+        readonly Stream                  stream;
+        readonly Channel<IZLoggerEntry>  channel;
+        readonly Task                    writeLoop;
+        readonly Task                    summaryWriteLoop;
+        readonly ZLoggerOptions          options;
         readonly CancellationTokenSource cancellationTokenSource;
 
         public AsyncStreamLineMessageWriter(Stream stream, ZLoggerOptions options)
         {
-            this.newLine = Encoding.UTF8.GetBytes(Environment.NewLine);
```

Comment:
stop committing these changes, it's really annoying. please revert all the lines you didn't actually change

Reply:
This is just the code formatter I use, this is too much to revert and redo the entire PR just for blank lines